gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -0,0 +1,178 @@
+"""
+This script builds a constraint pipeline that calculates constraint metrics.
+
+The constraint metrics pipeline will compute for LoF variants, missense variants, and
+synonymous variants:
+    - The number of observed variants
+    - The number of expected variants
+    - The observed: expected ratio
+    - The confidence interval around the observed: expected ratio
+    - pLI score (Probability of loss-of-function intolerance; probability that
```

Comment:
Just for curiousity, did you use black "--preview" as well for formatting docstrings? It doesn't format docstrings on my end.

Reply:
No, I tried to find something that would automatically do this for us, but couldn't so I think we will need to just do it ourselves.